includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -1909,10 +1927,10 @@ protected function guess_modal_role( DOMElement $modal ) {
 			return 'dialog';
 		}
 
-		$classes = $modal->getAttribute( 'class' );
+		$classes = explode( ' ', $modal->getAttribute( 'class' ) );
```

Comment:
This can be made more robust. Consider non-space whitespace and spaces at the beginning/end of an attribute value.

```suggestion
		$classes = preg_split( '/\s+/', trim( $modal->getAttribute( 'class' ) ) );
```